_data/chains/eip155-201030.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Alaya Dev Testnet",
+  "chain": "Alaya",
+  "network": "testnet",
+  "rpc": [
+    "https://devnetopenapi.alaya.network/rpc",
+	"wss://devnetopenapi.alaya.network/ws"
+  ],
+  "faucets": [
+  "https://faucet.alaya.network/faucet/?id=f93426c0887f11eb83b900163e06151c"
+  ],
+  "nativeCurrency": {
+    "name": "ATP",
+    "symbol": "atp",
+    "decimals": 18
+  },
+  "infoURL": "https://www.alaya.network/",
+  "shortName": "alaya",
+  "chainId": 201030,
+  "networkId": 1,
+  "icon": "alaya",
+  "explorers": [
+    {
+      "name": "alaya explorer",
+      "url": "devnetscan.alaya.network",
```

Comment:
the explorer URL should start with the protocol (e.g. https://)

Reply:
fixed
please take a look again, thank you very much!